build.gradle

```diff
@@ -117,6 +125,20 @@ targetCompatibility = 1.8
 
 mainClassName = 'org.terasology.launcher.TerasologyLauncher'
 
+// Swagger config
+swaggerSources {
+    client {
+        inputFile = file('swagger.json')
+        code {
+            language = 'java'
+            configFile = file('api-config.json')
+        }
+    }
+}
+generateSwaggerCodeClient.outputDir = file('web-api-client')
```

Comment:
ah, you can =) what is the structure of the generated content? is it a full java project?

Reply:
Yes, it's a complete gradle project. You can do `gradle idea` in the root project to regenerate idea files and then restart IntelliJ or open TerasologyLauncher.ipr to get code completion.